### PR TITLE
Add keyboard support.

### PR DESCRIPTION
**Pull request by kossel**
I'm not very familiar with testing, can you have a look and let me know it this works?

### CODE REVIEW

File: src/Panel.jsx [hunk 1]
```diff
@@ -5,12 +5,25 @@ import PanelContent from './PanelContent';
 import Animate from 'rc-animate';
 
 class CollapsePanel extends Component {
+  constructor(props) {
+    super(props);
+    this.handleItemClick = this.handleItemClick.bind(this);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+  }
+
   handleItemClick() {
```
Comment (afc163) [inline src/Panel.jsx]: Try `handleItemClick = () => {` instead of binding in constructor.

File: src/Panel.jsx [hunk 2]
```diff
@@ -35,12 +49,14 @@ class CollapsePanel extends Component {
       [`${prefixCls}-item-disabled`]: disabled,
     }, className);
     return (
-      <div className={itemCls} style={style} id={id} role="tablist">
```
Comment (afc163) [inline src/Panel.jsx]: Why remove `role="tablist"`?

**Comment by benjycui** [comment]
@afc163 实际使用起来的效果可以找 九向 review 一下。

File: src/Panel.jsx (hunk 2 above)
Comment (kossel) [reply src/Panel.jsx]: I just moved it down to an inner div, since what we want to focus would be the header instead of the whole panel.

**Comment by kossel** [comment]
what happen to this PR?

**Comment by goodmind** [comment]
is this going to be merged?

File: src/Panel.jsx [hunk 3]
```diff
@@ -35,12 +43,14 @@ class CollapsePanel extends Component {
...
+      <div className={itemCls} style={style} id={id} >
```
Comment (afc163) [inline src/Panel.jsx]:
```diff
- id={id} >
+ id={id}>
```

**Comment by afc163** [comment]
close #83

**Comment by afc163** [comment]
1.9.0 bumped

**Comment by enzoferey** [comment]
Hi, I'm having issues with redux-form Field component passed to "header" prop of Panel component. I can delete characters in the input but I can't write. Working in rc-collapse 1.8.0 but not in 1.9.0.

How should we proceed?

**Pull request merged**